README.md

```diff
@@ -37,6 +46,14 @@ A notebook can be turned into a public notebook by publishing a link to it. This
 If you want to run the copy yourself, or do interactive changes, you must download the notebook and reupload on your own account. Downloading the raw format of the previous example can be done by adding format=raw to the previous example …revisions-sql.ipynb?format=raw. This download-reupload-process is somewhat awkward.
 
 Note that a notebook will always be published, as the link can be guessed, so don't add any private information. 
+# chat
+ Applicants can get help and feedback from both mentors and community members. Community members discuss their contributions in a public chat.
```

Comment:
We hope not only GSoC/Outreachy applicants contribute to this project. You should rewrite this sentence as if we were inviting anyone to contribute.

Example:

"Contributors can get help and feedback from other community members in a public IRC channel. [...]"